### ISSUE TITLE
Bad error message not giving clue what parameter is wrongly configured.

### ISSUE THREAD

**Issue by sebas798** [MediaBrowser/Emby#2056]
When attempting to run a scan, the following error message appears.  Full log generated by the action is attached just below the snippet.

```
2016-08-14 05:38:40.7216 Error TaskManager: Error
    *** Error Report ***
    Version: 3.0.6060.0
    Command line: /usr/lib/emby-server/bin/MediaBrowser.Server.Mono.exe -programdata /var/lib/emby-server -ffmpeg /usr/bin/ffmpeg -ffprobe /usr/bin/ffprobe -restartpath /usr/lib/emby-server/restart.sh
    Operating system: Unix 4.4.0.34
    Processor count: 8
    64-Bit OS: True
    64-Bit Process: True
    Program data path: /var/lib/emby-server
    Mono: 4.2.1 (Debian 4.2.1.102+dfsg2-7ubuntu4)
    Application Path: /usr/lib/emby-server/bin/MediaBrowser.Server.Mono.exe
    Value cannot be null.
    Parameter name: path
    System.ArgumentNullException
```

[Emby server - "Value Cannot be null. Parameter name:  path" - Error message not helpful.txt](https://github.com/MediaBrowser/Emby/files/416979/Emby.server.-.Value.Cannot.be.null.Parameter.name.path.-.Error.message.not.helpful.txt)

**THIS BUG IS NOT ABOUT THE SCAN FAILING.**  It is about the fact that the error message is not helpful.  It does not tell the user what parameter is badly configured.  Instead of `Parameter name: path`, the actual configuration path name should show up.  The error should be caught earlier by the Emby Server code and proper error reporting should then be performed.

As a side note, if you do know how to solve my problem, you can contact me offline.

**Comment by LukePulverenti** [comment]
Please attach a complete server log, thanks.

**Comment by sebas798** [comment]
As requested, full log file generated with the following actions:

1. /etc/init.d/emby-server start
1. &lt;initiate full scan&gt;
1. /etc/init.d/emby-server stop

[server-63606774071.txt](https://github.com/MediaBrowser/Emby/files/417118/server-63606774071.txt)

Please notice the the previous attached file contains the output of the &lt;initiate full scan&gt; action.

**Comment by LukePulverenti** [comment]
Unfortunately the log doesn't indicate what path the problem is on. did you customize any server paths? does this happen with a fresh installation of the server on default settings?

**Comment by sebas798** [comment]
The fact that the log doesn't indicate the path is the reason of this bug report. :)  This bug report is about having the log generate a better, helpful message.  It is not about fixing/finding the cause of the error.  If the code can be fixed to generate a more verbose error, I could find the root cause.

That said, emby server worked flawlessly under Ubuntu 14.04 LTS.  When I upgraded to 16.04 LTS, the package was removed.  I reinstalled it from the official PPA and since, I get this error.  I cannot figure out why.
